_job-scheduler-plugin/build-plugin.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: Building a plugin for Job Scheduler
+nav_order: 10
+---
+
+# Buidling a plugin for Job Scheduler
+
+OpenSearch plugin developers can extend the Job Scheduler plugin to schedule jobs like running aggregation query against raw data and saving the aggregated data into a new index every hour, or continue monitoring the shard allocation by calling the OpenSearch API and then post the output to a Webhook.
+
+Reference the following [README](https://github.com/opensearch-project/job-scheduler/blob/main/README.md) for examples on building a plugin that utilizes Job Scheduler.
+
+## Defining an endpoint
+
+You can configure your plugin's API endpoint by referencing the [example](https://github.com/opensearch-project/job-scheduler/blob/main/sample-extension-plugin/src/main/java/org/opensearch/jobscheduler/sampleextension/SampleExtensionRestHandler.java) `SampleExtensionRestHandler.java` file. Set the `WATCH_INDEX_URI` to the endpoint name that your plugin will use:
```

Comment:
Change to: Set the endpoint url that your plugin will expose like `WATCH_INDEX_URI`